src/main/java/oracle/kubernetes/operator/PodWatcher.java

```diff
@@ -49,7 +48,6 @@
    * Factory for PodWatcher
    * @param ns Namespace
    * @param initialResourceVersion Initial resource version or empty string
-   * @param destination Callback for watch events
```

Comment:
I'm suspicious here...  Need to verify logic with you.  For Ingress, Service, and Domain, we have a single callback method, but for Pod we have two use cases: 1) general watch for add, change, delete as for other resource types, and 2) watch for Pod to come to its "Ready" state.  Obviously, these could be refactored into one method called by the PodWatcher, but I'm going to want to verify the behavior has been refactored correctly.

Reply:
Actually, you didn't change the functionality...  Just please restore the Javadoc.